docs/01-app/01-getting-started/11-deploying.mdx

```diff
@@ -4,40 +4,77 @@ nav_title: Deploying
+- [Replit](https://github.com/nextjs/deploy-replit)
+
+## Docker
+
+Next.js can be deployed to any provider that supports [Docker](https://www.docker.com/) containers. This includes container orchestrators like Kubernetes or a cloud provider that runs Docker.
+
+Docker deployments support all Next.js features. Learn how to [configure them](/docs/app/guides/self-hosting) for your infrastructure.
+
+### Templates
+
+- [Docker](https://github.com/vercel/next.js/tree/canary/examples/with-docker)
+- [Docker Multi-Environment](https://github.com/vercel/next.js/tree/canary/examples/with-docker-multi-env)
+- [DigitalOcean](https://github.com/nextjs/deploy-digitalocean)
+- [Fly.io](https://github.com/nextjs/deploy-fly)
+- [Google Cloud Run](https://github.com/nextjs/deploy-google-cloud-run)
 - [Render](https://github.com/nextjs/deploy-render)
 - [SST](https://github.com/nextjs/deploy-sst)
 
-Please refer to each provider's documentation for information on supported Next.js features.
+## Static export
 
-## Self-Hosting
+Next.js enables starting as a static site or [Single-Page Application (SPA)](/docs/app/guides/single-page-applications), then later optionally upgrading to use features that require a server.
 
-Self-hosting may mean you're responsible for provisioning your own servers, managing containers, and scaling. You can self-host Next.js in three different ways:
+Since Next.js supports [static exports](/docs/app/guides/static-exports), it can be deployed and hosted on any web server that can serve HTML/CSS/JS static assets. This includes tools like AWS S3, Nginx, or Apache.
 
-- [A Node.js server](/docs/app/guides/self-hosting#nodejs-server)
-- [A Docker container](/docs/app/guides/self-hosting#docker-image)
-- [A static export](/docs/app/guides/self-hosting#static-html-export)
+Running as a [static export](/docs/app/guides/static-exports) **does not** support Next.js features that require a server. [Learn more](/docs/app/guides/static-exports#unsupported-features).
 
-We have community maintained deployment examples with the following self-hosting providers:
+### Templates
 
-- [DigitalOcean](https://github.com/nextjs/deploy-digitalocean)
-- [Fly.io](https://github.com/nextjs/deploy-fly)
 - [GitHub Pages](https://github.com/nextjs/deploy-github-pages)
-- [Google Cloud Run](https://github.com/nextjs/deploy-google-cloud-run)
 
-> **🎥 Watch:** Learn more about self-hosting Next.js → [YouTube (45 minutes)](https://www.youtube.com/watch?v=sIVL4JMqRfc).
+## Adapters
+
+Next.js can be adapted to run on different platforms to support their infrastructure capabilities.
+
+Refer to each provider's documentation for information on supported Next.js features:
+
+- [AWS Amplify](https://docs.amplify.aws/nextjs/start/quickstart/nextjs-app-router-client-components)
+- [Cloudflare](https://developers.cloudflare.com/pages/framework-guides/nextjs/ssr/supported-features)
+- [Deno](https://docs.deno.com/examples/next_tutorial)
```

Comment:
nit: this section mentions "platforms", "infrastructure", and "provider", but Deno is none of these. Should this be about Deno Deploy? or maybe there should be a fifth heading (like there is one for Node.js), maybe "Deno" or "Alternative runtimes"

Reply:
Yeah, good callout. I'm specifically referencing Deno Deploy. Will update that.